EnglishFilter/sections/general_url.txt

```diff
@@ -12,6 +12,7 @@
 .com/pw/waWQiOjEw
 ! https://github.com/AdguardTeam/AdguardFilters/pull/83787
 ! ex. http://plqbxvnjxq92.com/gs6h79qcg?key=693ca25a821bb830696c0bf0ccd7c072
+/^http:\/\/img[a-z]+\.xyz(\/[a-z]{2,4}){2}\.js$/$script,~third-party
```

Comment:
As far as I can tell after checking, this seems normal, but please provide a list of examples from https://regex101.com/

Reply:
Oh, sorry I forgot. Is adding `ex. http://imgbigo.xyz/bb/cc.js` line enough?